api/src/test/java/org/openmrs/module/eptsreports/reporting/intergrated/calculation/pvls/InitialArtStartDateCalculationTest.java

```diff
@@ -0,0 +1,125 @@
+package org.openmrs.module.eptsreports.reporting.intergrated.calculation.pvls;
+
+import java.sql.Timestamp;
+import java.util.Arrays;
+import java.util.Collection;
+import org.junit.Assert;
+import org.junit.Test;
+import org.openmrs.Location;
+import org.openmrs.api.context.Context;
+import org.openmrs.calculation.patient.PatientCalculation;
+import org.openmrs.calculation.patient.PatientCalculationContext;
+import org.openmrs.calculation.result.CalculationResultMap;
+import org.openmrs.calculation.result.SimpleResult;
+import org.openmrs.module.eptsreports.reporting.calculation.pvls.InitialArtStartDateCalculation;
+import org.openmrs.module.eptsreports.reporting.intergrated.calculation.BasePatientCalculationTest;
+
+public class InitialArtStartDateCalculationTest extends BasePatientCalculationTest {
+
+  @Override
+  public PatientCalculation getCalculation() {
+    return Context.getRegisteredComponents(InitialArtStartDateCalculation.class).get(0);
+  }
+
+  @Override
+  public Collection<Integer> getCohort() {
+    return Arrays.asList(new Integer[] {2, 6, 7, 8, 999, 432});
+  }
+
+  @Override
+  public CalculationResultMap getResult() {
+    PatientCalculation calculation = getCalculation();
+    CalculationResultMap map = new CalculationResultMap();
+
+    // initiated ART by hiv enrolment
+    PatientCalculationContext evaluationContext = getEvaluationContext();
+    map.put(
+        2,
+        new SimpleResult(
+            new Timestamp(testsHelper.getDate("2008-08-01 00:00:00.0").getTime()),
```

Comment:
I can see this patient has encounters and obs defined but I don't see this particular result date value defined anywhere in calculationTest.xml. So in this case where is the initialArtStartDate of 2008-08-01 being defined for this patient?